spec/controllers/hunters_controller_spec.rb

```diff
@@ -27,8 +27,8 @@
       hunter: {
         name: 'Killua',
         playbook_id: create(:playbook).id,
-        harm: 8,
-        luck: 0
+        harm: nil,
+        luck: nil
```

Comment:
These values aren't what's making these attributes invalid. It's the absence of rating attributes like charm, cool, weird, etc.

Reply:
Reverted these back to their original values. 👍